build-system/server.js

```diff
@@ -30,6 +30,7 @@ const host = process.env.SERVE_HOST;
 const port = process.env.SERVE_PORT;
 const useHttps = process.env.SERVE_USEHTTPS == 'true' ? true : false;
 const gulpProcess = process.env.SERVE_PROCESS_ID;
+const quiet = process.env.SERVE_QUIET == 'true' ? true : false;
```

Comment:
this could have just been `const quiet = process.env.SERVE_QUIET == 'true';` (that already returns a boolean) right?